doc/tasks/php_parser.md

```diff
@@ -0,0 +1,73 @@
+# Phpparser
+
+The Phpparser task will test your changes for blacklisted or whitelisted keywords, such as `die()`, `var_dump()` etc.
+It lives under the `php_parser` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        php_parser:
+            visitors_options:
+              function_call_visitor:
+                blacklist: ["var_dump", "error_log"]
+                whitelist: ["count", "get_class"]
```

Comment:
What exactly is the difference between blacklist and whitelist. Aren't whitelist items allowed by default?
In the code I see that whitelist returns a notice. But the notice will still make the task fail?
I am not sure what this should do :)